src/libs/kata-sys-util/Cargo.toml

```diff
@@ -11,6 +11,7 @@ license = "Apache-2.0"
 edition = "2018"
 
 [dependencies]
+byteorder = "~1"
```

Comment:
Why use '~' instead of '^'

Reply:
Thanks for your review, change to `byteorder = "1.4.3"` for clearly.